### PR TITLE
Modal

### PR DESCRIPTION
**Pull request by khushijohri2001**
Please review my code for my Component Library's Modal Component.

The page includes:

Header with Girlifycoder Logo, Heading, and Github link
The navigation bar on the left
Sidebar on the right
Details about the Component and snippet

https://user-images.githubusercontent.com/65448358/160251059-be12752a-8979-4a40-85a3-06f69e7651a3.mp4

### CODE REVIEW

File: modals/styles2-modals.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+.child-one {
+  background-color: rgb(52, 158, 190);
+  padding: 8px;
+  color: white;
+  border-radius: 4px;
+  width: 40px;
+  text-align: center;
+}
+.child-two {
+  background-color: rgb(52, 158, 190);
```
Comment (pnchinmay) [inline modals/styles2-modals.css:61]: Use color variables

**Review by pnchinmay — APPROVED** [review]

File: modals/styles2-modals.css [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+.modal-container {
+  margin: 3rem;
+}
+
+.modal-main-heading {
+  color: yellow;
+}
+.modal-snippet iframe {
+  margin: 1rem 0rem;
+  width: 564px;
```
Comment (i-am-a-codegirl) [inline modals/styles2-modals.css:11]: use rems instead of px, its better for accomplishing responsiveness faster

**Review by i-am-a-codegirl — APPROVED** [review]
looks good to merge
Consider the suggestions

File: modals/styles2-modals.css (hunk 2 above)
Comment (khushijohri2001) [reply modals/styles2-modals.css:11]: ok

**Pull request merged**